schema.json

```diff
@@ -435,20 +435,7 @@
                   "$ref": "#/properties/services/items/properties/upstream/properties/timeout"
                 },
                 "vars": {
-                  "type": "array",
```

Comment:
I remember this schema file is synced from upstream? Do we need to apply changes to upstream as well?

Reply:
This is the true schema in APISIX, lua-resty-expr expressions are difficult to express in a simple way, maybe I'll try it later, but for now it's a simple fix.